pkg/common/k8s.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package common
```

Comment:
I'm not convinced that this is the right place for all of this, since it's a more specific logic than the simpler functions in this file.
I'd put it somewhere in the `platfrom/kube` package (not sure exactly where), so it can still be shared by the dashboard & controller.

Reply:
moved to platform/kube/utils/utils.go